src/main/webapp/app/course/tutorial-groups/tutorial-groups-instructor-view/tutorial-groups/crud/create-tutorial-group/create-tutorial-group.component.ts

```diff
@@ -27,11 +27,6 @@ export class CreateTutorialGroupComponent implements OnInit {
     }
 
     createTutorialGroup(formData: TutorialGroupFormData) {
-        // required fields
-        if (!formData?.title || !formData?.teachingAssistant) {
-            return;
-        }
```

Comment:
Is this no longer required?

Reply:
The form submission is disabled if the required fields are missing, so this is code duplication basically. Therefore I removed it